src/util/animate.ts

```diff
@@ -176,10 +178,6 @@ export function animate(
     return index > -1 && runningAnimations.splice(index, 1)[0];
```

Comment:
Actually I think it should be handled by the registry

Reply:
Rethinking
The context should hold a `cancelled` flag instead of the `cancel` variable
Then registry should expose `register(context): CancelFunction`
That makes more sense IMO